config/orte_check_lsf.m4

```diff
@@ -85,12 +85,21 @@ AC_DEFUN([ORTE_CHECK_LSF],[
                     [orte_check_lsf_happy="no"],
                     [orte_check_lsf_happy="yes"])
 
+          # liblsf requires shm_open, shm_unlink, which are in librt
+          OPAL_SEARCH_LIBS_COMPONENT([shm_open_rt], [shm_open], [rt],
+                        [shm_open_rt_happy="yes"],
+                        [shm_open_rt_happy="no"])
```

Comment:
Any particular reason you didn't just set `orte_check_lsf_happy` to `yes` or `no`?  I.e., why the additional `AS_IF`?

Reply:
It matches the `yp_all_nsl_happy` pattern above it. That's the only reason.

Reply:
If you don't mind, could you consolidate?  No need to propagate a sub-optimal pattern from the past.

Thanks!

Reply:
Done. Take a look.